iaac/terraform/apps/profiles-and-kfam/main.tf

```diff
@@ -1,7 +1,12 @@
+data "github_repository_file" "profile_controller_policy" {
+  repository = "awslabs/kubeflow-manifests"
```

Comment:
Preferred to use Git hash here as well to avoid any break in the future.

Reply:
We can't use tags yet (https://github.com/integrations/terraform-provider-github/issues/1517) but we can use a branch. I made the same comment above to get feedback whether we should rely on "their" branch not changing or whether we should use our "ni" fork instead.